lib/gooddata/models/model.rb

```diff
@@ -188,7 +188,7 @@ def upload_multiple_data(data, project_blueprint, options = { :client => GoodDat
                   end
                 end
               else
-                csv_header = File.open(path, &:gets).split(',')
+                csv_header = File.open(path, &:gets).chomp('').split(',')
```

Comment:
why is passing an empty string to chomp needed?
"test\n".chomp => "test"
"test\n".chomp '' => "test"